src/app.js

```diff
@@ -455,10 +455,13 @@ var run = function () {
 
   function editorOnChange () {
     var input = editor.getValue()
+    // save contents
+    editor.setCacheFileContent(input)
```

Comment:
If saving every time the file is modified is too slow it could save on tab close. Also, all files would need to be saved when any file is compiled because of imports.